easygui/boxes/diropen_box.py

```diff
@@ -43,6 +43,9 @@ def diropenbox(msg=None, title=None, default=None):
     title = ut.getFileDialogTitle(msg, title)
     localRoot = tk.Tk()
     localRoot.withdraw()
+    localRoot.lift()
```

Comment:
This solved my issue on Windows 10. It *also* solved the same issue with:

1. fileopen_box
2. filesave_box